examples/basic/android/app/build.gradle

```diff
@@ -132,8 +136,8 @@ android {
 
 dependencies {
     implementation project(':react-native-video')
-    implementation "androidx.appcompat:appcompat:1.0.0"
     implementation "com.facebook.react:react-native:+"  // From node_modules
+    implementation 'org.webkit:android-jsc:+'
```

Comment:
where is it used?

Reply:
this is part of the RN 0.60 template. I'm hoping to simplify some of this in the future but was trying to do the minimum possible to get the example running.